tictactoe/eacs/src/test/kotlin/com/anaplan/engineering/azuki/tictactoe/analysis/PlayOrder.kt

```diff
@@ -1,21 +1,29 @@
 package com.anaplan.engineering.azuki.tictactoe.analysis
 
 import com.anaplan.engineering.azuki.core.runner.AnalysisScenario
-import com.anaplan.engineering.azuki.tictactoe.adapter.api.TicTacToeImplementation
 import com.anaplan.engineering.azuki.tictactoe.dsl.TicTacToeScenario
 import com.anaplan.engineering.azuki.tictactoe.eacs.O
 import com.anaplan.engineering.azuki.tictactoe.eacs.X
 import com.anaplan.engineering.azuki.tictactoe.eacs.gameA
+import org.junit.runners.Parameterized
 
-class PlayOrder : TicTacToeScenario() {
+class PlayOrder(private val player1: String, private val player2: String) : TicTacToeScenario() {
```

Comment:
I presume this means that the parametrisation can only apply to the whole test class, so one cannot mix scenarios with different parametrisation, or just non-parametrised and parametrised scenarios. Is this correct?

Reply:
that's the junit way